jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/LifeCycleBaseTest.java

```diff
@@ -2388,7 +2390,9 @@ public void beforeTaskStartedEvent(TaskEvent event) {
 
             @Override
             public void beforeTaskCompletedEvent(TaskEvent event) {
-                assertNull(event.getTask().getTaskData().getTaskInputVariables());                
+                assertNotNull(event.getTask().getTaskData().getTaskInputVariables());
+                assertEquals(1, event.getTask().getTaskData().getTaskInputVariables().size());
+                assertTrue(event.getTask().getTaskData().getTaskInputVariables().containsKey("input"));                
```

Comment:
correct, due to complete task command is loading variables they will already be available in the beforeTaskCompleted event. I don't think this requires docs update unless it's already there saying there won't be any data. Anyway, this is not guaranteed as one listener can load it meaning all other will have the data in place.